ext/bcmath/libbcmath/src/bcmath.h

```diff
@@ -106,7 +106,13 @@ void bc_int2num(bc_num *num, int val);
 
 long bc_num2long(bc_num num);
 
-int bc_compare(bc_num n1, bc_num n2);
+typedef enum {
+	EQUAL = 0,
+	LEFT_GREATER = 1,
+	RIGHT_GREATER = -1
```

Comment:
The case names should probably be prefixed, given they effectively act as global constants.

Reply:
Good point!

Reply:
Done.